airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/PostgresUtils.java

```diff
@@ -18,12 +18,21 @@ public class PostgresUtils {
 
   public static PgLsn getLsn(final JdbcDatabase database) throws SQLException {
     // pg version >= 10. For versions < 10 use query select * from pg_current_xlog_location()
-    final List<JsonNode> jsonNodes = database
-        .bufferedResultSetQuery(conn -> conn.createStatement().executeQuery("select * from pg_current_wal_lsn()"),
-            resultSet -> JdbcUtils.getDefaultSourceOperations().rowToJson(resultSet));
+    return getLsnImpl(database, "pg_current_wal_lsn");
+  }
+
+  public static PgLsn getInsertLsn(final JdbcDatabase database) throws SQLException {
```

Comment:
Do we have any test with pgVer < 10

Reply:
I don't think so, but I can easily (and gladly) add that. Which versions of pg should we support?